src/me/EtienneDx/RealEstate/Transactions/TransactionsStore.java

```diff
@@ -21,307 +30,685 @@
 import me.EtienneDx.RealEstate.Messages;
 import me.EtienneDx.RealEstate.RealEstate;
 import me.EtienneDx.RealEstate.Utils;
-import me.ryanhamshire.GriefPrevention.Claim;
-import me.ryanhamshire.GriefPrevention.GriefPrevention;
+import me.EtienneDx.RealEstate.ClaimAPI.IClaim;
 
-public class TransactionsStore
-{
-    public final String dataFilePath = RealEstate.pluginDirPath + "transactions.data";
-    DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
-    Date date = new Date();
+public class TransactionsStore {

```

Comment:
It would be nice to have an interface and different implementations instead of everything in here